scripts/revng

```diff
@@ -224,35 +159,6 @@ def split_dash_dash(args):
     return list(extra_args), args
 
 
-def get_architecture(path):
```

Comment:
Who does this now?

Reply:
in liftPipe.cpp
```
  BinaryFile TheBinary(SourceBinary.path()->str(), BaseAddress);
  TheBinary.architecture().name()
```
and then it's later read from the model when used
lift should store it in the model i think